packages/evolution-generator/src/tests/test_labels_generator.py

```
@@ -106,6 +108,168 @@ def test_get_labels_file_path_widgets():
     assert path == "../../example/demo_generator/fr/section1.yaml"
 
 
+# Path where create_mocked_excel_data writes the workbook; we delete it after each test.
+MOCKED_EXCEL_FILE = "src/tests/references/test.xlsx"
+
+
+class TestAddTranslationsFromExcel:
+    """Tests for add_translations_from_excel function"""
+
+    @pytest.mark.parametrize(
+        "headers, rows, expected_translations_fr, expected_translations_en",
```

Ruff (0.15.12)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)